src/slurmmail/cli.py

```diff
@@ -764,6 +765,18 @@ def __process_spool_file(
         msg["From"] = options.email_from_address
         msg["Date"] = email.utils.formatdate(localtime=True)
         msg["Message-ID"] = email.utils.make_msgid()
+
+        # add optional headers
+        if options.email_headers:
+            for header_name, header_value in options.email_headers.items():
+                if header_name in msg:
+                    logger.warning(
```

Comment:
This should be `logging.warning` as Slurm-Mail currently uses the "root" logger rather than its own (for now).

Reply:
Also, please change to:

```python
logging.warning("Ignoring header_name '%s' - header has already been set", header_name)
```

Ref: https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/logging-fstring-interpolation.html